internal/config/default_config.go

```diff
@@ -66,6 +66,11 @@ var (
 				Name:      "daemonsets.v1.apps",
 				PublishTo: "meshery.meshsync.core",
 			},
+			//Added Ingress support
+			{
+   				Name:      "ingresses.v1.networking.k8s.io",
```

Comment:
```suggestion
   				Name:      "ingresses.v1.",
```

It'd be good to capture Ingresses a little more broadly.